### ISSUE TITLE
Contents in vision_benchmark/resources?

### ISSUE THREAD

**Issue by tsly123** [sIncerass/MVLPT#3]
Hi,

Thank you for sharing your work.
I have a question regarding the contents of `MVLPT/trainers/vision_benchmark/resources/` which are `datasets, knowledge, model` folders.
Are these files (e.g. `knowledge/gpt3/GPT3_mnist.tsv`) needed for the evaluation? If yes, could you share the information on how to generate them?

I ran an evaluation 5-shot on PathMnist (size 28x28, 9 classes) (in MedMnist) dataset which is quite similar to Patch-Camelyon reported in the paper. The results are less than 0.22 accuracy. I don't know the reason.

**Comment by bronyayang** [comment]
Hi,

For everything in vision_benchmark folder, we took from ELEVATOR benchmark [https://github.com/Computer-Vision-in-the-Wild/Elevater_Toolkit_IC](url). I don't think we use anything other than its constructing dataset code. I don't think the files in resources is needed for evaluation.

Regarding the performance, can you provide more details, such as the accuracy is compared against what?

**Comment by tsly123** [comment]
Hi @bronyayang ,

The low accuracy is my mistake. I've fixed it already.

However, I have 2 other questions.

1. Since I can't run the `main_single_elevator_cut.sh` (in [here](https://github.com/sIncerass/MVLPT/issues/1#issuecomment-1424982940)), I run `bash scripts/mvlpt/main_single_coopdata_cut.sh VPT vit_b16 4 5 1 medmnistdataset` instead. The medmnistdataset is not in CoOp or ELEVATER data.

My question is does the `main_single_coopdata_cut.sh` perform few-shot finetuning as the `main_single_elevator_cut.sh`?

2. The difference between shots. Below are the log file of 1_shot and 20_shot with the same trainer VPT, same seed1, and same dataset. `TRAIN BATCH_SIZE` is 32 (as default in `https://github.com/sIncerass/MVLPT/blob/main/configs/trainers/MVLPT/vit_b16.yaml`)
What is the meaning of this difference? 

- Log file of 1 and 5 shots. Note **batch [1/1]**
```
epoch [1/200] batch [1/1] time 1.385 (1.385) data 1.319 (1.319) loss 3.0781 (3.0781) acc 0.0000 (0.0000) lr 2.0000e-03 eta 0:04:35
Evaluate on the *val* set
Overall evaluation !
=> result
* total: 17,778
* correct: 1,807
* accuracy: 10.2%
* error: 89.8%
* macro_f1: 3.4%
results OrderedDict([('accuracy', 10.164247946900664), ('error_rate', 89.83575205309934), ('macro_f1', 3.3893807375442426)])
```
- Log file of 20 shots. Note **batch [5/6]**
```
epoch [1/200] batch [5/6] time 0.055 (0.322) data 0.000 (0.250) loss 3.0664 (3.2238) acc 15.6250 (10.6250) lr 1.0000e-05 eta 0:06:25
Evaluate on the *val* set
Overall evaluation !
=> result
* total: 17,778
* correct: 1,819
* accuracy: 10.2%
* error: 89.8%
* macro_f1: 2.6%
results OrderedDict([('accuracy', 10.23174710316121), ('error_rate', 89.76825289683879), ('macro_f1', 2.6310290636368396)])
```

**Comment by tsly123** [comment]
Solved!!!
Thank you
